prow/config/config.yaml

```diff
@@ -644,7 +640,7 @@ presubmits:
     - name: build
       branches:
         - main
-      run_if_changed: '^api|^test|^Makefile$'
+      run_if_changed: "^api|^test|^Makefile$"
```

Comment:
single quotes have been used consistently in the file afaik, so please keep it aligned

Reply:
/hold

to resolve ^ comment

Reply:
This is weird editor glitch. It changes this quote only when auto-formatting, but doesn't touch any of the other quotes. I think Kashif's prow config change has same thing. Well, fixed now.